standard/flags_master_data.lua

```diff
@@ -2120,10 +2125,12 @@ local aliases = {
 	['esmx'] = 'spanishspeaking',
 	['ptbr'] = 'portuguesespeaking',
 	['ruby'] = 'russianspeaking',
+	['arab'] = 'arabicspeaking',
 
 	--language flag aliases
 	['engspeaking'] = 'englishspeaking',
 	['gerspeaking'] = 'germanspeaking',
+	['arabstates'] = 'arabicspeaking',
```

Comment:
don't see why this should be a valid alias

Reply:
That's what it is currently called when used accross multiple wikis. It is less than 50 pages though, so can easily be changed.